scripts/python/phdf_diff.py

```diff
@@ -26,15 +26,16 @@
 def Usage():
     print("""
 
-    Usage: %s [-quiet] [-brief] [-all] [-one] [--tol=eps] file1.phdf file2.phdf
+    Usage: %s [-quiet] [-brief] [-all] [-one] [--tol=eps] [-ignore_metadata] file1.phdf file2.phdf
 
-           -all: report all diffs at all positions
-           -one: Quit after first different variable
-         -brief: Only report if files are different
-                 Overrides --all
-         -quiet: Only report if files are different and
-                 don't print any extraneous info.
-      --tol=eps: set tolerance to eps.  Default 1.0e-12
+                  -all: report all diffs at all positions
+                  -one: Quit after first different variable
+                -brief: Only report if files are different
+                        Overrides --all
+                -quiet: Only report if files are different and
+                        don't print any extraneous info.
+             --tol=eps: set tolerance to eps.  Default 1.0e-12
+      -ignore_metadata: Ignore differences in metadata
```

Comment:
I don't expect any changes, but do you know why some of these are using double dashes and others are not is this a convention? I always thought the double dashes were for the long version and single dashes for short versions of the same argument, it looks like here the double dash prefix is being used only in cases where an assignment is involved. Could probably create an issue (low priority), this should probably be uniform between our python scripts.

Reply:
I also don't see any consistency here. I think `./tst/regression/run_test.py` is the only other python script to do argument parsing.